src/Strategist/BuildOrders/DefaultBuildOrders.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include<utility>
+#include<BWAPI.h>
+
+extern std::initializer_list<std::pair<BWAPI::UnitType, int>> unknown_smallest;
```

Comment:
The `std::initializer_list` can be defined in their headers, just make them const.